pkg/controller/chi/worker.go

```diff
@@ -253,7 +253,7 @@ func (w *worker) updateCHI(old, new *chop.ClickHouseInstallation) error {
 		WithStatusAction(new).
 		M(new).F().
 		Info("reconcile started")
-	w.a.V(2).M(new).F().Info("action plan\n%s\n", actionPlan.String())
+	//w.a.V(2).M(new).F().Info("action plan\n%s\n", actionPlan.String())
```

Comment:
Was it intentional to disable this log line? (If so: why not remove it completely?)

Reply:
When debugging, this will report null pointer exception. I wonder if it is an environment problem, so do this.